pulsar-broker/pom.xml

```diff
@@ -164,6 +164,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.streamnative.oxia</groupId>
+      <artifactId>oxia-testcontainers</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Should you remove this dependency? There is no new test added to `pulsar-broker`.